src/core/EnvelopeAndLfoParameters.cpp

```diff
@@ -380,12 +378,22 @@ void EnvelopeAndLfoParameters::loadSettings( const QDomElement & _this )
 /*	 ### TODO:
 	Old reversed sustain kept for backward compatibility
 	with 4.15 file format*/
+
 	if( _this.hasAttribute( "sus" ) )
 	{
 		m_sustainModel.loadSettings( _this, "sus" );
 		m_sustainModel.setValue( 1.0 - m_sustainModel.value() );
 	}
 
+	// ### TODO:
+/*	// Keep compatibility with version 2.1 file format
+	if( _this.hasAttribute( "lfosyncmode" ) )
+	{
+		m_lfoSpeedKnob->setSyncMode(
+		( TempoSyncKnob::TtempoSyncMode ) _this.attribute(
+						"lfosyncmode" ).toInt() );
+	}*/
+
```

Comment:
Did you mean to leave this commented? Also curious if you meant to put `1.2` and not `2.1`.